scientific_library/tvb/simulator/models/infinite_theta.py

```diff
@@ -0,0 +1,871 @@
+from tvb.simulator.models.base import Model
+from tvb.basic.neotraits.api import NArray, List, Range, Final
+
+import numpy
+
+
+class MontbrioPazoRoxin(Model):
+    r"""
+    2D model describing the Ott-Antonsen reduction of infinitely all-to-all
+    coupled QIF neurons (Theta-neurons) as in Montbrio, Pazo, Roxin, 2015.
+
+    The two state variables :math:`r` and :math:`V` represent the average
+    firing rate and the average membrane potential of our QIF neurons.
+
+    The equations of the infinite QIF 2D population model read
+
+    .. math::
+            \dot{r} &= 1/tau (Delta/(pi tau) + 2 V r),
```

Comment:
why is there a `,` at the end of the line? (have this question in other model docstrings too)

Reply:
This is an artifact of copy-pasting the docstring from other model. Do we have a quick way to render these?

Reply:
Ok, Spyder seems to be a decent tool to edit the docstrings.